docs/using-the-compiler.rst

```diff
@@ -199,6 +199,13 @@ Output Description
             // The Ethereum Contract ABI. If empty, it is represented as an empty array.
             // See https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI
             abi: [],
+            // See the Metadata Output documentation (serialised JSON string)
+            metadata: "{...}",
+            // User documentation (natspec)
+            userdoc: {},
+            // Developer documentation (natspec)
+            devdoc: {},
+            // EVM-related outputs
```

Comment:
Since the new IR is still not part of the compiler and indeed it seems to be different, it would make sense moving it out of of the `evm` object.